test/exports/Export.test.ts

```diff
@@ -0,0 +1,19 @@
+// This test suite is used to generate test cases to be exported to other implementations and is not run as part of
```

Comment:
I was confused by the export thing at first but I think this comment helps. I'm not sure how I feel about it tbh, still maybe don't fully get it - is it being used for anything right now?

Reply:
Yeah I'm not super happy about this either, as its co-opting a test system to run tooling/scripts and I am very open to a better solution. 

The test cases are used to detect breaking changes in the node software. As this library is currently the canonical reference implementation, known good requests are exported and run against the node software to ensure the end-to-end functionality. This acts as a 'reverse integration' test of sorts. 

Options:
- Run it as a external script :: would require adding additional dependencies( ts-node ) 
- Move into separate repo 
- Remove the tests entirely.

Reply:
I've removed the TestCase exporting as needs more work. Its presence is largely counter productive and adds more confusion than anything. If needed we can revisit.